app/services/support_interface/candidate_journey_tracker.rb

```diff
@@ -155,7 +155,7 @@ def received_references_timings
       @received_reference_timings ||=
         received_references.map do |reference|
           {
-            received_at: earliest_update_audit_for(reference, feedback_status: 'feedback_provided'),
+            received_at: reference.feedback_provided_at,
```

Comment:
:+1: together with the extra `includes` this is likely to be a bit faster (because it avoids a 1+n query). It could lead to a higher memory usage.